### DISCUSSION TITLE
use table orchid in blade

### DISCUSSION THREAD

**Discussion by itpmsoftware** [orchidsoftware/platform#2402] [General]
Hello team,
I have a question, Can I use the table component inside the blade template?

![image](https://user-images.githubusercontent.com/65979963/195640396-77ecf1ce-5d5e-4194-9d72-c8ec0c663252.png)

I want to choose my options and after clicking on the button show the result, Does anybody have any idea how to do that?

**Comment by ValerianMemsk** [comment]
Hello. 
Look at this topic: https://github.com/orchidsoftware/platform/discussions/2386
I suppose it would be similar.

**Reply by itpmsoftware** [reply]
Did you use a table in the blade? I want an example, I was trying but always show the error when try to implement a table
